src/core/public/chrome/ui/header/collapsible_nav.test.tsx

```diff
@@ -143,14 +138,12 @@ describe('CollapsibleNav', () => {
       mockRecentNavLink({ label: 'recent 1' }),
       mockRecentNavLink({ label: 'recent 2' }),
     ];
-    const customNavLink = mockLink({ title: 'Custom link' });
     const component = mount(
       <CollapsibleNav
         {...mockProps()}
         isNavOpen={true}
         navLinks$={new BehaviorSubject(navLinks)}
         recentlyAccessed$={new BehaviorSubject(recentNavLinks)}
-        customNavLink$={new BehaviorSubject(customNavLink)}
```

Comment:
This line should not be deleted I think?

Reply:
`customNavLink$` is defined in mockProps